grammar.js

```diff
@@ -649,6 +680,36 @@ module.exports = grammar({
       $._statement,
     ),
 
+    // This is missing binary expressions, others were kept so that macro code can be parsed better and code examples
+    _top_level_expression_statement: $ => seq(
```

Comment:
same here, we could factorize things by using
a _expression_no_binary_exprs intermediate rule.